deployment/main.tf

```diff
@@ -329,4 +338,32 @@ resource "google_kms_key_ring" "firmware_release_prod" {
 #   }
 # }
 
+resource "google_kms_key_ring" "terraform_state" {
+  name = "armored-witness-bucket-tfstate"
+  location = "europe-west2"
```

Comment:
would it be a good idea to make this into a `var.state_location` and reuse in the `google_storeage_bucket` below too? (just thinking that if we decided to move state elsewhere, but accidentally only moved one of these two things, we'd be doubling our exposure to unavailability risk)

Reply:
Done.